system/i18n/formbuilder/item-types/csrf_token.properties

```diff
@@ -0,0 +1,5 @@
+title=CSRF Token
+iconclass=fa-shield red
+description=CSRF token protection
+
+invalid.csrf_token.error.message=Invalid CSRF Token. Please try submitting your form again.
```

Comment:
This message should be less techy - I think we should have examples of CSRF messages elsewhere that we can use. But basically something that doesn't mention "CSRF". Perhaps:

> Your form submission has been prevented due to a security check failure. This may be caused by a timeout. Please review your form entries and resubmit.

^^ something _like_ that.

Reply:
Updated to what was already in core!